net48/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("RazorpayClient48")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("RazorpayClient48")]
+[assembly: AssemblyCopyright("Copyright ©  2024")]
```

Comment:
```suggestion
[assembly: AssemblyTitle("Razorpay")]
[assembly: AssemblyDescription("Razorpay SDK for .Net")]
[assembly: AssemblyConfiguration("")]
[assembly: AssemblyCompany("Razorpay Software Private Limited")]
[assembly: AssemblyProduct("Razorpay.API")]
[assembly: AssemblyCopyright("Copyright © 2015 - 2024")]
```
Should't it be this? Correct me if mistaken.

Reply:
updated 

https://github.com/razorpay/razorpay-dot-net/blob/10148cd90cfb84ff83d6f4e8643488f5fe22eef5/net48/Properties/AssemblyInfo.cs#L8